tests/tests/util/testWithMoonbeam.ts

```diff
@@ -93,14 +94,10 @@ export async function startMoonbeamNode(
     binary.stdout.on("data", onData);
   });
 
-  const polkadotJsTypes = require("../../../polkadot-js/standalone-types.json");
-  const polkadotJsRpc = require("../../../polkadot-js/frontier-rpc-types");
-
   const wsProvider = new WsProvider(`ws://localhost:${WS_PORT}`);
   const polkadotApi = await ApiPromise.create({
     provider: wsProvider,
-    types: polkadotJsTypes,
-    rpc: polkadotJsRpc,
+    typesBundle,
```

Comment:
So we don't need the RPC stuff anymore because is it all stock frontier which means Jaco included it already, right?

Dumb question: why isn't it `types: typesBundle`?

Reply:
No such thing as a dumb question!

types field is for non-versioned types, i.e. will provide the same types for all moonbeam versions.
typesBundle provides custom types depending on the version